src/jobtypes.py

```diff
@@ -68,11 +67,13 @@ def __init__(self, repo_name='mozilla-inbound'):
                 #  opt test web-platform-tests-4"
                 if job['build_system_type'] == 'buildbot':
                     testtype = job['ref_data_name'].split(' ')[-1]
+                    buildername = job['ref_data_name']
 
                 # taskcluster's testtype is a part of its 'job_type_name' like reftest-2
                 # for [TC] Linux64 reftest-2
                 else:
                     testtype = job['job_type_name'].split(' ')[-1]
+                    buildername = job['job_type_name']
```

Comment:
hmm, what's the meaning of **agnostic to both systems** ? Is that mean we should create a new variable name which doesn't belong to Buildbot and TaskCluster either? How about **job_label**? I always feel lost with variable naming.